test/contracts/staking/staking-factory-events.ts

```diff
@@ -0,0 +1,48 @@
+import {BigNumber, Event} from 'ethers'
+import {StakingPoolCreatedEvent} from '../../../typechain-types/StakingPoolFactory'
+import {expect} from 'chai'
+import {StakingPoolType} from './staking-events'
+import {isAddress} from 'ethers/lib/utils'
+
+type RewardToken = {
+    token: string
+}
+
+export type ActualStakingPoolCreatedEvent = {
+    // stakingPool: string
```

Comment:
nit; commented out line

question: is that field needed, should it be part of the verify?

Reply:
remove

Reply:
keep in actual, remove in expect